package/cpp/rnskia/values/RNSkValue.h

```diff
@@ -28,6 +28,13 @@ class RNSkValue : public RNSkReadonlyValue
   ~RNSkValue() {
     unsubscribe();
   }
+
+  JSI_HOST_FUNCTION(cancelAnimation) {
```

Comment:
Alternatively this change could be removed and we would provide the following function instead:
```tsx
const cancelAnimation = (value: Value) => {
  if (value.animation) {
    value.animation.cancel();
  }
};
```